File: ISL.ReIdentification.WebApps.Portals/ISL.ReIdentification.Portals.Client/src/models/ReIdentifications/IdentificationRequest.ts
import { IdentificationItem } from "./IdentificationItem";

export class IdentificationRequest {
    public id: string = crypto.randomUUID();
    public identificationItems: IdentificationItem[] = [];
    public displayName: string = "";
    public surname?: string = "";
    public givenName?: string = "";
    public email: string = "";
    public jobTitle?: string = "";
    public organisation?: string = "";
    public reason: string = "";
}

File: ISL.ReIdentification.WebApps.Portals/ISL.ReIdentification.Portals.Server.Tests.Acceptance/Models/ReIdentifications/IdentificationRequest.cs
// ---------------------------------------------------------
// Copyright (c) North East London ICB. All rights reserved.
// ---------------------------------------------------------

using System;
using System.Collections.Generic;

namespace ISL.ReIdentification.Portals.Server.Tests.Acceptance.Models.ReIdentifications
{
    public class IdentificationRequest
    {
        public Guid Id { get; set; }
        public List<IdentificationItem> IdentificationItems { get; set; }
        public string EntraUserId { get; set; }
        public string GivenName { get; set; }
        public string Surname { get; set; }
        public string DisplayName { get; set; }
        public string JobTitle { get; set; }
        public string Email { get; set; }
        public string Organisation { get; set; }
        public string Reason { get; set; }
    }
}
